src/test/java/Homework17.java

```diff
@@ -0,0 +1,33 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.annotations.Test;
+public class Homework17 extends BaseTest {
+
+@Test
+    public void addSongToPlaylist() throws InterruptedException {
+    navigateToPage();
+    login("demo@class.com", "te$t$tudent");
+    Thread.sleep(2000);
+    grabASong();
+}
+
+public void grabASong() {
+    WebElement song = driver.findElement(By.xpath("//article[@data-test='song-card']"));
+    WebElement playlist = driver.findElement (By.xpath("//section[@id='playlists']//li[3]"));
+
+    Actions acts = new Actions(driver);
+    acts.clickAndHold(song)
+            .release(playlist)
+            .build()
+            .perform();
+
+    playlist.click();
+   // acts.dragAndDrop(song, playlist).build().perform();
+
+
```

Comment:
Please finish the homework 17!

Please, **validate that there is a song added.**

____________________________________________________________________________________________________________________________

1. Create a new file Homework17.java with @Test annotation in IntelliJ IDEA.

2. Create a test case addSongToPlaylist() using @Test annotations
and the helper/reusable methods we created.

3. Navigate to "https://bbb.testpro.io/".

4. Log in with your credentials.

5. Add a song to a playlist.

6. Validate that there is a song added.

7. Create a new branch and commit your changes.

8. Push your code to a remote repository.

9. Create a pull request.